packages/cli/src/lib/setups/i18n/replacers.ts

```diff
@@ -196,133 +196,6 @@ export async function replaceServerI18n(
   });
 }
 
-/**
- * Adds I18nLocale import and pass it to Storefront<I18nLocale> type as generic in `remix.env.d.ts`
- */
-export async function replaceRemixEnv(
```

Comment:
delete this because the default type for I18n in the generic is `{ language: LanguageCode; country: CountryCode }` and thats the only thing this replacer adds